storage.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"os"
+
+	"cloud.google.com/go/storage"
+	"golang.org/x/net/context"
+)
+
+var (
+	client     *storage.Client
+	bucket     *storage.BucketHandle
+	bucketName string
+)
+
+func init() {
+	var err error
+	client, err = storage.NewClient(context.Background())
+	if err != nil {
+		panic(err)
```

Comment:
This happens when the server starts, not on the request. This will only panic if the server can't connect to google cloud storage when the server starts